Docs: Getting started edits (#19915)
grafana/grafana: docs/sources/installation/mac.md

type = "docs"
[menu.docs]
parent = "installation"
weight = 500
+++

> # Installing on Mac

Reviewer: ```suggestion
# Installing on macOS
```

> This page provides instructions to help you install Grafana on your Mac system. You can either install using homebrew or a binary tar file.

Reviewer: ```suggestion
This page provides instructions to help you install Grafana on macOS. You can either install using homebrew or a binary tar file.
```

Default login and password `admin`/ `admin`

> 

Author: All removed content was moved to configuration.md. If it is generic rather than Mac-specific, then we should remove the "Mac"

`conf` folder and override any of the settings defined in
`conf/defaults.ini`.

### Start Grafana

> Start Grafana by executing `./bin/grafana-server web`. The `grafana-server` binary needs the working directory to be the root install directory (where the binary and the `public` folder is located).

Reviewer: ```suggestion
Start Grafana by executing `./bin/grafana-server`. The `grafana-server` binary needs the working directory to be the root install directory (where the binary and the `public` folder is located).
```
